src/orders/utils/returns.test.ts

```diff
@@ -0,0 +1,161 @@
+import { DiscountValueTypeEnum, OrderLineFragment } from "@dashboard/graphql";
+import { FormsetData } from "@dashboard/hooks/useFormset";
+
+import { LineItemData } from "../components/OrderReturnPage/form";
+import { getItemPriceAndQuantity } from "./data";
+
+const mockedOrderLines: Array<OrderLineFragment & { orderLineId?: string }> = [
```

Comment:
question: Do you need all fields in this mock, maybe you could use only the necessary, The second question shouldn't be this in fixtures?